planning/work-items/analysis/analysis-manageable-affordances.md

```diff
@@ -13,31 +15,134 @@ However, the following points are not supported:
+- As an astronomer I want to instruct a digital telescope to capture a series of images of a section of the night sky over a six hour period (another example with an output)
+
+### Queue of Actions
+
+- **Who:** Thing Description designer
+- **What:** Describe a queue of actions as a resource, including its contents and state
+- **Why:** So that I can fully describe the current status of a device capable of running persistent actions
+- **Sentence:** As a Thing Description designer, I need to describe a queue of actions as a resource so that I can fully describe the current status of a device capable of running monitorable and cancelable actions.
+- **Process Stakeholders**:
+  - Submitter: WoT WG/extended community
+  - Specification Writers:
+  - Implementation Volunteers:
+  - Impacted People: TD Designers, Consumer application developers.
+  - Impact Type: Increase complexity of the WoT interaction model. This implies an increase in implementation complexity of both Consumer and Exposer applications.
+- **Linked Use Cases or Categories:**
+  - https://github.com/w3c/wot-thing-description/issues/1070
+  - https://github.com/w3c/wot-thing-description/issues/1779
+
+#### Scenarios
+
+- As a customer using a ticket kiosk I want to tell the printer to print three tickets and notify me on the screen when it is complete
+- As a factory worker I want to tell a robotic arm to make a series of movements in sequence, without having to wait for each movement to complete before starting the next
+- As a factory worker I want to tell a 3D printer to print a series of objects so that I don't have to wait for one object to be completed before submitting the next
+
+### Cuncurrent Actions
+
+- **Who:** Thing Description designer
+- **What:** Describe actions that can be executed mutliple times concurrently
+- **Why:** So that I can describe devices that can handle multiple running operations at the same time
+- **Sentence:** As a Thing Description designer, I need to describe actions that can be executed mutliple times concurrently so that I can describe devices that can handle multiple running operations at the same time.
+- **Process Stakeholders**:
+  - Submitter: WoT WG/extended community
+  - Specification Writers:
+  - Implementation Volunteers:
+  - Impacted People: TD Designers, Consumer application developers.
+  - Impact Type: Increase complexity of the WoT interaction model. This implies an increase in implementation complexity of both Consumer and Exposer applications.
+
+#### Scenarios
+
+- As an office worker using a high end photocopier I want to copy multiple multi-page documents at the same time to save me time
+- As a factory worker I want to instruct a multi-head 3D printer to print two objects at once so that I don't have to wait for one to finish before starting the other
```

Comment:
```suggestion
- As a factory worker I want to instruct a multi-head 3D printer to print two objects at once so that I don't have to wait for one to finish before starting the other
- As two office workers using a high-end automated coffee machine with two brew chambers, we want to brew two coffees at the same time to avoid waiting time
```